### DISCUSSION TITLE
Any plan to add the description of beatmaps from beatmap listing tab?

### DISCUSSION THREAD

**Discussion by KiwiKin1** [ppy/osu#22850] [Q&A] [answered]
https://user-images.githubusercontent.com/106151046/224555243-bbcd5273-d26a-4bd7-aa65-35f44f560d7e.mp4

![image](https://user-images.githubusercontent.com/106151046/224555373-9ea1c97e-9dc4-4e28-b9b9-1e36898885b9.png)

as the video and image above, any plan for future implementation of description of beatmaps?

**Comment by cdwcgt** [comment, marked as answer]
This needs to solve bbcode parsing, and may not be considered for the time being, I remember that the priority of this problem is not high

**Reply by Joehuu** [reply]
Also already tracked at https://github.com/ppy/osu/issues/8023.
